modules/ROOT/pages/apikit-odatav4-extension-reference.adoc

```diff
@@ -218,50 +206,51 @@ Default configuration
 
 === For Configurations
 
-* <<Config>> {nbsp}
+* <<Config>> 
 
 === Throws
 
-* APIKIT-ODATA:INTERNAL_SERVER_ERROR {nbsp}
-* APIKIT-ODATA:MALFORMED_SEARCH {nbsp}
-* APIKIT-ODATA:MALFORMED_URI {nbsp}
-* APIKIT-ODATA:NOT_IMPLEMENTED {nbsp}
-* APIKIT-ODATA:TRANSFORMATION {nbsp}
-
+* APIKIT-ODATA:INTERNAL_SERVER_ERROR 
+* APIKIT-ODATA:MALFORMED_SEARCH 
+* APIKIT-ODATA:MALFORMED_URI 
+* APIKIT-ODATA:NOT_IMPLEMENTED 
+* APIKIT-ODATA:TRANSFORMATION 
```

Comment:
Ping @JuanAller do we have a new error type for errors during expansion?

Reply:
Awaiting for devs answer. Thanks!

Reply:
We have to add `APIKIT-ODATA:EXPANSION`.

Reply:
Added! Thanks!